pkg/cmd/cli/cmd/tag.go

```diff
@@ -133,7 +133,7 @@ func parseStreamName(defaultNamespace, name string) (string, string, error) {
 
 func determineSourceKind(f *clientcmd.Factory, input string) string {
 	mapper, _ := f.Object()
-	gvks, err := mapper.KindsFor(schema.GroupVersionResource{Group: imageapi.GroupName, Resource: input})
+	gvks, err := mapper.KindsFor(schema.GroupVersionResource{Group: imageapi.LegacyGroupName, Resource: input})
```

Comment:
The mapper did not know `istag`. Haven't deeply investigate. But this list only lists core group shorthands: https://github.com/openshift/origin/blob/9949be492fe910aa1778032e445ad9b197b60e7a/vendor/k8s.io/kubernetes/pkg/kubectl/kubectl.go#L159. Maybe, we have to add our groups? Also compare the matching algorithm: https://github.com/openshift/origin/blob/9949be492fe910aa1778032e445ad9b197b60e7a/vendor/k8s.io/kubernetes/pkg/kubectl/cmd/util/shortcut_restmapper.go#L175